.github/workflows/build-push-image.yml

```diff
@@ -246,3 +265,76 @@ jobs:
             --platform \
             linux/${{ matrix.platform.name }} \
             ${{ needs.merge.outputs[format('digest_{0}', matrix.platform.name)] }}
+
+  sign:
+    if: ${{ inputs.sign }}
+    runs-on: "ubuntu-latest"
+    env:
+      COSIGN_PASSWORD: "password"
+      COSIGN_YES: true
```

Comment:
I'm wondering, why are these two envvars needed, and should they be documented somewhere?

For instance, I assume that `cosign --yes` is equivalent to `COSIGN_YES=true`. Also, does the test key have a password called "password"?